eclass/unpacker.eclass

```diff
@@ -340,6 +340,12 @@ _unpacker() {
 	local m=$(echo "${a}" | tr '[:upper:]' '[:lower:]')
 	a=$(find_unpackable_file "${a}")
 
+	# do nothing with release signatures
```

Comment:
We're going to need to get review for this from @mgorny and post it to the gentoo-dev mailing list (for eclass changes, unless he feels it's trivial).

Reply:
Sure! Who will post it?